source/4.0/guides/sql-how-to/joins-and-complex-where.html.md

```diff
@@ -0,0 +1,54 @@
+---
+chapter: SQL How To
+title: Use complex where clauses with joins
+---
+
+This article assumes:
+
+* You have a `users` table
+* You have a `posts_users` table 
+
+Let's imagine that you have some Users and each has many Posts associated. You decided to implement this relation 
```

Comment:
I would use another example. It's hard to grasp why we would like to get unassociated posts with the user's one.

Why not make something following our own restriction implementation ? 
I'll draft something tomorrow with you